tsl/src/continuous_aggs/options.c

```diff
@@ -80,9 +80,21 @@ cagg_find_groupingcols(ContinuousAgg *agg, Hypertable *mat_ht)
 {
 	List *retlist = NIL;
 	ListCell *lc;
-	Oid cagg_view_oid =
-		get_relname_relid(NameStr(agg->data.user_view_name),
-						  get_namespace_oid(NameStr(agg->data.user_view_schema), false));
+	Oid cagg_view_oid;
+
+	/*
+	 * Get the direct_view definition for the finalized version because
```

Comment:
Could you also update the comment above the function to clarify that we now have 2 forms for caggs (since 2.7) and how they differ ? (the comment now refers to the old style)

Reply:
Updated.